mkinitcpio/riscv64.patch

```diff
@@ -1,36 +0,0 @@
-Index: PKGBUILD
-===================================================================
---- PKGBUILD	(revision 454173)
-+++ PKGBUILD	(working copy)
-@@ -20,20 +20,27 @@
- provides=('initramfs')
- backup=('etc/mkinitcpio.conf')
- source=("https://sources.archlinux.org/other/$pkgname/$pkgname-$pkgver.tar.gz"{,.sig}
--       0001-Fix-the-warning-about-missing-modules.builtin.modinf.patch)
-+       0001-Fix-the-warning-about-missing-modules.builtin.modinf.patch
-+       mkinitcpio-generic-gzip-kernel.patch::https://github.com/archlinux/mkinitcpio/pull/112.patch
```

Comment:
https://github.com/archlinux/mkinitcpio/pull/112 doesn’t appear in any releases.

Reply:
But the package is already building successfully, do we still need to keep the patch?

Reply:
But the package is already building successfully, do we still need to keep the patch?

Reply:
> But the package is already building successfully, do we still need to keep the patch?

This affects building initramfs, instead of building `mkinitcpio` itself.